docs/sources/configuration/_index.md

```diff
@@ -660,28 +661,39 @@ alertmanager_client:
 
 # Use DNS SRV records to discover Alertmanager hosts.
 # CLI flag: -ruler.alertmanager-discovery
+# Deprecated: Use `alertmanagers_per_tenant` instead
 [enable_alertmanager_discovery: <boolean> | default = false]
 
 # How long to wait between refreshing DNS resolutions of Alertmanager hosts.
 # CLI flag: -ruler.alertmanager-refresh-interval
+# Deprecated: Use `alertmanagers_per_tenant` instead
 [alertmanager_refresh_interval: <duration> | default = 1m]
 
 # If enabled, then requests to Alertmanager use the v2 API.
 # CLI flag: -ruler.alertmanager-use-v2
+# Deprecated: Use `alertmanagers_per_tenant` instead
 [enable_alertmanager_v2: <boolean> | default = false]
 
 # List of alert relabel configs
+# Deprecated: Use `alertmanagers_per_tenant` instead
 alert_relabel_configs:
   [- <relabel_config> ...]
 
 # Capacity of the queue for notifications to be sent to the Alertmanager.
 # CLI flag: -ruler.notification-queue-capacity
+# Deprecated: Use `alertmanagers_per_tenant` instead
 [notification_queue_capacity: <int> | default = 10000]
 
 # HTTP timeout duration when sending notifications to the Alertmanager.
 # CLI flag: -ruler.notification-timeout
+# Deprecated: Use `alertmanagers_per_tenant` instead
 [notification_timeout: <duration> | default = 10s]
 
+# Configure alertmanager per tenant.
+# A map with tenant ID as key.
+alertmanagers_per_tenant: 
```

Comment:
This doesn't really fit with the way we configure tenant-specific settings in Loki.
https://grafana.com/docs/loki/latest/configuration/#limits_config

Could you refactor this to fit the existing model please?

Reply:
Hey @dannykopping, thanks for the comment, I think you're right but that means we leave the existing config as a global config that is overridden per tenant in the limits config right?

Reply:
Correct, I think that's the only way we can support it right now that is consistent with the rest of Loki's configuration.